dbms/src/TiDB/Schema/SchemaBuilder.cpp

```diff
@@ -317,68 +459,63 @@ void SchemaBuilder<Getter, NameMapper>::applySetTiFlashReplica(DatabaseID databa
         }
 
         applyDropTable(db_info->id, table_id);
+        return;
```

Comment:
The changes in `applySetTiFlashReplica` is adding `return` instead of nesting `if ... else ...`. Make code more clear and simple.